app/src/test/java/com/github/multimatum_team/multimatum/util/QRGeneratorUtilityTest.kt

```diff
@@ -0,0 +1,38 @@
+package com.github.multimatum_team.multimatum.util
+
+import android.graphics.Bitmap
+import com.google.zxing.*
+import com.google.zxing.common.HybridBinarizer
+import com.google.zxing.qrcode.QRCodeReader
+
+/**
+ * Interface that contain utility methods use in QRGeneratorTest and QRGeneratorActivityTest
+ */
+interface QRGeneratorUtilityTest {
```

Comment:
Instead of defining an interface where all methods are already implemented, you could just write two functions at the toplevel.

Reply:
Well that's right. At begining i thought it would be a good idea to make an interface because these function are used by only 2 classe.